infra/terraform/dev/deploy/task-definitions.tf

```diff
@@ -0,0 +1,178 @@
+module "cache" {
+  source = "github.com/cloudposse/terraform-aws-ecs-container-definition?ref=0.46.0"
+
+  container_name   = "cache"
+  container_image  = var.ecs_cache_image
+  container_memory = var.ecs_cache_memory
+  container_cpu    = var.ecs_cache_cpu
+  essential        = true
+
+  log_configuration = {
+    logDriver = "awslogs"
+    options = {
+      "awslogs-group"         = "ecs/cache-${var.environment}"
+      "awslogs-region"        = var.region
+      "awslogs-stream-prefix" = "ecs"
+    }
+  }
+}
+
+module "database" {
+  source = "github.com/cloudposse/terraform-aws-ecs-container-definition?ref=0.46.0"
+
+  container_name   = "database"
+  container_image  = "${data.aws_ssm_parameter.DB_ECR_URL.value}:develop"
+  container_memory = var.ecs_db_memory
+  container_cpu    = var.ecs_db_cpu
+  essential        = true
+
+  log_configuration = {
+    logDriver = "awslogs"
+    options = {
+      "awslogs-group"         = "ecs/database-${var.environment}"
+      "awslogs-region"        = var.region
+      "awslogs-stream-prefix" = "ecs"
+    }
+  }
+}
+
+module "php-fpm" {
+  source = "github.com/cloudposse/terraform-aws-ecs-container-definition?ref=0.46.0"
+
+  container_name   = "php-fpm"
+  container_image  = "${data.aws_ssm_parameter.WEB_ECR_URL.value}:develop"
```

Comment:
Not blocking: May be worth making a variable for the repo branch name.

Reply:
I need to work on this in a future PR once the GH Actions piece works. This piece should only be needed one time and then GH Actions will be dynamically updating the image to use with the (SHA) tag.